pywinauto/controls/HwndWrapper.py

```diff
@@ -163,6 +163,10 @@ class HwndWrapper(object):
     can_be_label = False
     has_title = True
 
+    # specify whether we need to grab an image of ourselves
+    # when asked for properties
+    _NeedsImageProp = False
```

Comment:
@airelil, `_NeedsImageProp = False` in base class used as a default value, expect it would not be changed in subclasses only reassigned directly or via `@property`.I see no reason to use here @property statement. Here is an expected behavior - https://gist.github.com/moden-py/8b62c9e2f7d08bd11474.